src/models/notarizedHealthCert/createNotarizedHealthCert/createUnwrappedHealthCert.ts

```diff
@@ -30,14 +26,14 @@ export const createUnwrappedDocument = (
     url: "https://healthcert.renderer.moh.gov.sg/",
   };
 
-  const notarisationMetadata: NotarisationMetadata = {
+  const notarisationMetadata: notarise.NotarisationMetadata = {
     reference,
     notarisedOn: dateString,
     passportNumber: passportNumber || "",
     url: storedUrl,
   };
-  if (euHealthCertQr) {
-    notarisationMetadata.signedEuHealthCert = euHealthCertQr.qrData;
+  if (signedEuHealthCerts) {
+    notarisationMetadata.signedEuHealthCerts = signedEuHealthCerts;
   }
```

Comment:
Can help to add one more test to verify that `notarisationMetadata.signedEuHealthCerts` will not be defined when there are no signEuHealthCerts? (i.e. it should never be an empty `[]`)